import-service/serverless.yml

```diff
@@ -32,6 +53,12 @@ functions:
           path: import
           method: get
           cors: true
+          authorizer:
+            name: tokenAuthorizer
+            arn: "arn:aws:lambda:us-east-1:241876787615:function:authorization-service-dev-basicAuthorizer"
```

Comment:
Hardcode? I'd recommend to obtain this `arn` programmatically. There are at least couple of ways how to refer to this lambda =)

Reply:
Agree, for example:
![image](https://user-images.githubusercontent.com/33042066/100789920-2b04c080-3428-11eb-998c-7b13624994de.png)